packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -277,6 +278,10 @@ export class Calendar extends React.Component<ICalendarProps, ICalendarState> im
     }
   }
 
+  private _onBlur = (event: React.FocusEvent<HTMLDivElement>) => {
+    event.stopPropagation();
```

Comment:
you need to call the original props.onBlur here. otherwise anything passed by user from `onBlur` prop is an no-op because you are entirely overwriting it with your own logic here